ImageItem/ImageItem.js

```diff
@@ -113,16 +131,16 @@ const ImageItemBase = kind({
 	},
 
 	computed: {
-		className: ({captionPosition, styler}) => styler.append({
-			captionOverlay: captionPosition === 'overlay'
-		})
+		className: ({captionPosition, styler, sizing}) => styler.append({captionOverlay: captionPosition === 'overlay'}, sizing)
 	},
 
-	render: ({captionPosition, children, css, disabled, orientation, src, ...rest}) => {
+	render: ({captionPosition, children, css, disabled, orientation, skin, src, ...rest}) => {
 		const [Component, marqueeProps] = (children && (orientation === 'horizontal' || captionPosition === 'below')) ? [MarqueeImageItem, {
-			alignment: 'center'
+			alignment: skin === 'silicon' && orientation === 'horizontal' ? 'left' : 'center'
 		}] : [UiImageItem, null];
 
+		delete rest.sizing;
```

Comment:
Is there any reason to not pass `sizing` prop to 'Component'? Internally, 'Image' component is used for an image and it already supports `sizing` prop via `ui/Image` component. We can set margin for 'silicon' skin, but it looks that `sizing` needs to be handled by `ui/Image` in general.

Reply:
I reverted the change, the initial motive was to not affect the other skins as they did not have sizing specifications.